### DISCUSSION TITLE
Jemalloc error when running container

### DISCUSSION THREAD

**Discussion by Bayzon88** [qdrant/qdrant#5366] [Q&A]
Hi everyone, i've been getting this error recently when trying to run it on a raspberry pi with Pi OS 
<jemalloc>: Unsupported system page size
<jemalloc>: Unsupported system page size
memory allocation of 56 bytes failed

I know running qdrant on a raspberry pi is not ideal, i'm doing this for learning purposes. Please advice :)

**Comment by timvisee** [comment]
Out of curiosity, are you running on an 32-bit system?

Otherwise building [from source without jemallloc](https://github.com/orgs/qdrant/discussions/4989#discussioncomment-10495819) may do the trick. Based on your comment there you're already trying that.

**Reply by Bayzon88** [reply]
Hey there, i'm using a 64 bit system as i have an 8GB raspberry pi. I will try building from source thanks :)

**Comment by youroldmangaming** [comment]
https://hub.docker.com/r/y0mg/qdrant-raspberry-pi

**Comment by haktan-suren** [comment]
if your page size is 16, then use this https://hub.docker.com/r/haktansuren/qdrant-pi5-fixed-jemalloc

I could not make /dashboard work with @youroldmangaming version so had to build from scratch and made multi-stage

**Reply by sankemax** [reply]
@haktan-suren can you publish the related Dockerfile for the image you've built please?
